src/intents.js

```diff
@@ -108,6 +108,16 @@ function injectService (url, element, intent, data, onReadyCallback) {
 }
 
 const first = arr => arr && arr[0]
+// In a far future, the user will have to pick the desired service from a list.
+// For now it's our job, an easy job as we aribtrary pick the first service of
```

Comment:
typo s/aribtrary/arbitrary

Reply:
Isn't that comment out of date though ? The first service is the default choice now, but the `filterServices` already allows you to pick one, right?

Reply:
As the `filterServices` is a callback given as parameter, it's not the user which pick the service, the choice is still made by code.